agent/stats/engine.go

```diff
@@ -292,11 +292,6 @@ func (engine *DockerStatsEngine) addContainer(dockerID string) {
 		return
 	}
 
-	if err != nil {
```

Comment:
I wonder what was this originally intended for!

Reply:
It was an error check for https://github.com/aws/amazon-ecs-agent/commit/a8d27bcd1a992f099cb339f032e3b5768a3bf46c#diff-92abb112df31ca87deae8bec53c0a901L287.

Reply:
Yeah, was just looking at that from the commit message :)